sympy/core/power.py

```diff
@@ -1212,6 +1212,22 @@ def _is_one(expr):
                 or self.base.is_irrational):
                 return self.exp.is_rational
 
+    def _eval_is_transcendental(self):
+        # Gelfond-Schneider theorem
```

Comment:
Isn't this already implemented in `_eval_is_algebraic`? The assumptions system already knows that `transcendental == complex & !algebraic`.

Reply:
Where is this exactly?
```
>>> (4 + 5*I).is_transcendental
False
```
Complex values with rational real/imag give False in current master.

Reply:
Yes, looks like the remaining values are correct in master.
```
>> ((S(2)/3 + I*S(7)/8)**sqrt(2)).is_transcendental
True
>> ((1 + sqrt(3))**sqrt(2)).is_transcendental
True
```
So, retaining transcendental for exp for the time being.